experimental/packages/otlp-transformer/package.json

```diff
@@ -70,6 +68,7 @@
     "webpack": "4.46.0"
   },
   "dependencies": {
+    "@opentelemetry/api-metrics": "0.27.0",
```

Comment:
Is it temporary like with "sdk-metrics-base" package?
Should we add a similar "todo" entry here as well?

Reply:
I removed the ToDo in sdk-metrics-base in this PR.

I think the api-metrics package is temporary in general as it should be included in api once stable.
As mentioned above there are two variants:
1. api-metrics into peer dep like api
1. api-metrics as dependency like other core packages

I went for (2) as it was that way at most places and it seems better to me as long as api-metrics is in the same lerna project.

Reply:
api-metrics package is likely to break from version to version, before the initial stable version. I think it would be easier for us to declare api-metrics as a dependency like other core packages, instead of peer deps.

Reply:
I agree, keep it as dependency with pinned version and let it disappear into api once stable.

Reply:
Cool, thanks for the clarification